Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -308,14 +308,13 @@ void NetworkProcessProxy::getNetworkProcessConnection(WebProcessProxy& webProces
     RELEASE_LOG(ProcessSuspension, "%p - NetworkProcessProxy is taking a background assertion because a web process is requesting a connection", this);
     startResponsivenessTimer(UseLazyStop::No);
     NetworkProcessConnectionParameters parameters;
-    parameters.allowTestOnlyIPC = webProcessProxy.allowTestOnlyIPC();
 #if ENABLE(IPC_TESTING_API)
     parameters.ignoreInvalidMessageForTesting = webProcessProxy.ignoreInvalidMessageForTesting();
 #endif
     // The WebProcess shouldn't request a connection to the network process before we've had a chance to
```

Comment:
I think this comment is outdated now? At least we dropped the assertion below that enforced it.

Reply:
Will remove.